libs/ui/layout/src/lib/value-list/value-list.component.html

```diff
@@ -0,0 +1,44 @@
+<div
+  class="flex items-center justify-center"
+  cdkOverlayOrigin
+  #trigger="cdkOverlayOrigin"
+  [class]="extraClass"
+>
+  <gn-ui-button
+    #iconElement
+    type="light"
+    extraClass="flex items-center bg-transparent border-none justify-center"
+    (buttonClick)="toggleOverlay()"
+  >
+    <ng-icon name="iconoirList" size="24"></ng-icon>
+  </gn-ui-button>
+
+  <ng-template
+    cdkConnectedOverlay
+    [cdkConnectedOverlayOrigin]="trigger"
+    [cdkConnectedOverlayOpen]="isOpen"
+    [cdkConnectedOverlayPositions]="[overlayPosition]"
+    cdkConnectedOverlayPush
+    [cdkConnectedOverlayWidth]="'auto'"
+    [cdkConnectedOverlayFlexibleDimensions]="true"
+    [cdkConnectedOverlayGrowAfterOpen]="true"
+    (detach)="close()"
+  >
+    <div class="bg-white shadow-lg border border-gray-300 flex">
```

Comment:
There are icons even when there is no value to display
![image](https://github.com/user-attachments/assets/39106fdb-151e-44ff-8912-738183e62db0)